modules/lz/sub-vending/main.bicep

```diff
@@ -0,0 +1,479 @@
+targetScope = 'managementGroup'
+
+// METADATA - Used by PSDocs
+
+metadata name = '`main.bicep` Parameters'
+
+metadata description = '''These are the input parameters for the Bicep module: [`main.bicep`](./main.bicep)
+
+This is the orchestration module that is used and called by a consumer of the module to deploy a Landing Zone Subscription and its associated resources, based on the parameter input values that are provided to it at deployment time.
+
+> For more information and examples please see the [wiki](https://github.com/Azure/bicep-lz-vending/wiki)'''
+
+// PARAMETERS
+
+// Subscription Parameters
+@metadata({
+  example: true
+})
+@sys.description('''Whether to create a new Subscription using the Subscription Alias resource. If `false`, supply an existing Subscription's ID in the parameter named `existingSubscriptionId` instead to deploy resources to an existing Subscription.
+
+- Type: Boolean
```

Comment:
I think we should try to get rid of anything that is only here for PSDocs. Thoughts?

Reply:
Yup I'm mulling this over hence leaving this PR in draft for now until I finish everything and make a final call. 👍

I have considered migrating to brm to generate the module docs in modules home repo. But I actually really like the per parameter verbose output and the way it creates the parameter file example based on parameter metadata in PSdocs. 

But I think for now I'll just have to maintain 2 copies of the module docs and make some manual changes if we need to cut a release. Mainly because the tests on this repo run a brm and if there it is a diff it will fail. So that pretty much means I have to move to brm generated docs.

I'm happy to go this way for now and happy to participate, provide feedback on changes and assist wherever possible. 

Happy to chat as always 👍👍

Reply:
have moved to brm docs here :)